Dockerfile

```diff
@@ -54,20 +90,36 @@ RUN go get -v ./...
 RUN mkdir caddy && cd caddy
 COPY go.mod go.sum ./
 
-RUN go get -v ./... && \
-    cd ..
+RUN go get -v ./...
 
 COPY . .
 
+# todo: automate this?
+# see https://github.com/docker-library/php/blob/master/8.2-rc/bullseye/zts/Dockerfile#L57-L59 for php values
+ENV CGO_LDFLAGS="-lssl -lcrypto -lreadline -largon2 -lcurl -lonig -lz $PHP_LDFLAGS" CGO_CFLAGS=$PHP_CFLAGS CGO_CPPFLAGS=$PHP_CPPFLAGS
```

Comment:
This is suspicious and we should figure out why this is required to link (maybe it belongs in a configuration file somewhere)? Or is this fine?

Reply:
This looks ok to me. In go we usually don't use Makefiles, this is the way to replace it.